.pylintrc

```diff
@@ -215,7 +215,7 @@ ignore-mixin-members=yes
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
 # Ignoring ansible.constants to suppress `no-member` warnings
-ignored-modules=ansible.constants
+ignored-modules=ansible.constants,distutils
```

Comment:
This should fix it. The problem is that the distutils namespace gets modified in runtime so pylint can't check it properly